core/src/main/java/com/klaytn/caver/contract/ContractIOType.java

```diff
@@ -100,4 +115,49 @@ public void setType(String type) {
     public void setIndexed(boolean indexed) {
         this.indexed = indexed;
     }
+
+    /**
+     * Setter function for components
+     * @param components The tuple's components list.
+     */
+    public void setComponents(List<ContractIOType> components) {
+        this.components = components;
+    }
+
+    /**
+     * Returns a type string.
+     * If type string has a tuple, the components of the tuple are listed and returned.
+     * @return String
+     */
+    public String getTypeAsString() {
+        if(getType().contains("tuple")) {
+            String typeString = getComponentAsString();
+            return getType().replace("tuple", "tuple" + typeString);
+        } else {
+            return getType();
+        }
+    }
+
+    private String getComponentAsString() {
+        if(this.getComponents() == null) {
+            return "";
+        }
+
+        StringBuilder stringBuilder = new StringBuilder("(");
+
+        for(int i=0; i<this.getComponents().size(); i++) {
+            ContractIOType ioType = this.components.get(i);
+            if(ioType.getType().contains("tuple")) {
+                stringBuilder.append(ioType.getTypeAsString());
+            } else {
+                stringBuilder.append(ioType.getType());
+            }
```

Comment:
```suggestion
            stringBuilder.append(ioType.getTypeAsString());
```
How about just calling `getTypeAsString` function?